index.html

```diff
@@ -0,0 +1,108 @@
+</head>
+
+<body class="body">
+    <div >
+        <div class="top">
+            <h1 class="ROSE"><img width="235px" height="30px" src="rose.png" alt=""></h1>
+            <div class="wholeBurger">
+                <div class="topPartOfTheBurger"></div>
+                <div class="middlePartOfTheBurger"></div>
+                <div class="bottomPartOfTheBurger"></div>
+            </div>
+        </div>
+        <div>
+            <img class="topPhoto" width="375px" height="235px"
+                src="imgonline-com-ua-Resize-urNhT9kARoZ94SZ.jpg">
+        </div>
+        <div class="textAfterTop">
+            <div class="aboutBlog">A few words about this blog platform, Ghost, and how this site was made</div>
+            <div class="smallTextAboutBlog">Why Ghost (& Figma) instead of Medium, WordPress or other options?</div>
+        </div>
+        <div class="lineOfFirstPage"></div>
+    </div>
+    <div class="secondPage">
+        <div class="articles">All articles</div>
+        <div>
+            <img class="secondPagePhotosFIRST" width="335px" height="194px"
+                src="imgonline-com-ua-Resize-bAyRCik3GxZyxPP7.jpg" alt="">
+            <div class="OFFF" >Granny gives everyone the finger, and other tips from OFFF Barcelona</div>
+        </div>
+        <div>
+            <img class="secondPagePhotos" width="335px" height="194px"
+                src="imgonline-com-ua-Resize-XOzt5UX5xp2iXq.jpg" alt="">
+            <div class="helloWorld">Hello world, or, in other words, why this blog exists</div>
+        </div>
+        <div>
+            <img class="secondPagePhotos" width="335px" height="194px"
+                src="imgonline-com-ua-Resize-wwIuzf7bbgI.jpg" alt="">
+            <div class="regarding">Here are some things you should know regarding how we work </div>
+        </div>
+        <div>
+            <img class="secondPagePhotos" width="335px" height="194px"
+                src="imgonline-com-ua-Resize-T3EhyoGpwJ.jpg" alt="">
+            <div  class="updatingList">Updating list of 50+ sources on distributed teams,
+                remote work, and how to make it all work better</div>
+        </div>
+        <div>
+            <img class="secondPagePhotos" width="335px" height="194px"
+                src="imgonline-com-ua-Resize-aZU0MMuMOA5pq.jpg" alt="">
+            <div class="fewWordd">A few words about this blog platform, Ghost, and how this
+                site was made</div>
+        </div>
+        <div>
+            <img class="secondPagePhotos" width="335px" height="194px"
+                src="imgonline-com-ua-Resize-X8Lnt3rhtf0Q.jpg" alt="">
+            <div class="modern" >How modern remote working tools get along with Old
+                School Cowboy's methods
+            </div>
+        </div>
+        <div class="blackSquare">
+            <div class="squareFirstText">
+
+                <div class="textCenter" class="DIGITAL" class="boldText">DIGITAL PRODUCT DESIGN&nbsp;</div>
```

Comment:
```suggestion
                <div class="text-center digital bold-text">DIGITAL PRODUCT DESIGN&nbsp;</div>
```
fix here and everywhere else